apps/coscrad-frontend/src/store/slices/id-generation/hooks/use-loadable-generated-id.tsx

```diff
@@ -0,0 +1,10 @@
+import { useLoadable } from '../../resources/shared/hooks';
+import { selectLoadableGeneratedId } from '../selectors';
+import { acquireId } from '../thunks';
+
+export const useLoadableGeneratedId = () =>
+    useLoadable({
+        selector: selectLoadableGeneratedId,
+        // TODO Rename this prop `thunk`
```

Comment:
Consider this.

Reply:
Command generation is a rare violation of CQS in our system. It does `fetch` some information (generated ID`, albeit via a `POST`. I think the name is ok.